manifests/snmp_port.pp

```diff
@@ -22,16 +22,14 @@
 #   Order can be used to configure where in `squid.conf`this configuration section should occur.
 define squid::snmp_port (
   Variant[Pattern[/\d+/], Integer]
-                    $port           = $title,
+  $port           = $title,
```

Comment:
While it's breaking the API, I wonder if we can do something like
```puppet
Stdlib::Port $port = Integer($title),
```

Reply:
I think we should do this in another PR with some more type cleanup

Reply:
Yes, different PR.